includes/namespace.php

```diff
@@ -41,7 +41,7 @@
  *
  * @param bool $network_wide Whether to activate network-wide.
  */
-function activate( bool $network_wide = false ): void {
+function activate( ?bool $network_wide = false ): void {
```

Comment:
Changed this because it's not always a bool, e.g. when activating a plugin via WP-CLI.

See https://github.com/wp-cli/extension-command/issues/348